HouseRules_Essentials/Rulesets/DifficultyHardRuleset.cs

```diff
@@ -15,8 +15,9 @@ internal static Ruleset Create()
                 description,
                 new CardEnergyFromAttackMultipliedRule(0.8f),
                 new CardEnergyFromRecyclingMultipliedRule(0.8f),
-                new EnemyHealthScaledRule(1.2f),
-                new EnemyAttackScaledRule(1.2f));
+                new EnemyHealthScaledRule(1.8f),
+                new EnemyAttackScaledRule(1.5f),
+                new LevelPropertiesModifiedRule("BigGoldPileChance", 30, "FloorOneHealingFountains", 1, "FloorOneLootChests", 2, "FloorTwoHealingFountains", 1, "FloorTwoLootChests", 2, "FloorThreeHealingFountains", 1, "FloorThreeLootChests", 1));                
```

Comment:
Values are passed into the `LevelPropertiesModifiedRule` rule in a _slightly_ different way.  Here's an example from The Swirl ruleset.  Note that values are defined more like pairs than all in a single line separated by commas:

https://github.com/orendain/DemeoMods/blob/118ad9a30668ce51d8211bfb89f520aa8377b4df/HouseRules_Essentials/Rulesets/TheSwirlRuleset.cs#L68-L76

Reply:
Ah. I see whatcha mean. Will correct and re-submit.